### ISSUE TITLE
Breaks after updating to 12.1.0

### ISSUE THREAD

**Issue by jaagupku** [terrestris/react-geo-ws#24]
Previous dependencies
```
    "@terrestris/react-geo": "^11.4.4",
    "antd": "^3.26.6",
    "ol": "^5.3.3",
```

Updated dependencies
```
    "@terrestris/react-geo": "^12.1.0",
    "antd": "^3.26.6",
    "ol": "^6.1.1",
```

React version: `"react": "^16.12.0"`

Workshop application stops working.

With minimal map component and layers zoom is broken
```
import React from 'react';
import { useBoolean } from 'react-hanger';

import 'ol/ol.css';
import 'antd/dist/antd.css';
import './iceMap.css';

import OlMap from 'ol/Map';
import OlView from 'ol/View';
import OlLayerTile from 'ol/layer/Tile';
import OlSourceOsm from 'ol/source/OSM';
import TileWMS from 'ol/source/TileWMS';

import { Drawer } from 'antd';
import {
  SimpleButton,
  MapComponent,
  MapProvider,
  mappify,
} from '@terrestris/react-geo';

const Map = mappify(MapComponent);

const osmBaseLayer = new OlLayerTile({
  source: new OlSourceOsm(),
});

const center = [788453.4890155146, 6573085.729161344];
const view = new OlView({
  center: center,
  zoom: 4,
});
const map = new OlMap({
  view: view,
  layers: [osmBaseLayer, eestiWMSLayer],
});

map.on('postcompose', map.updateSize);

function App(props) {
  const isDrawerVisible = useBoolean(false);

  return (
    <div className="App">
      <MapProvider map={map}>
        <Map />
        <SimpleButton
          style={{ position: 'fixed', top: '30px', right: '30px' }}
          onClick={isDrawerVisible.toggle}
          icon="bars"
        />
        <Drawer
          title="react-geo-application"
          placement="right"
          onClose={isDrawerVisible.toggle}
          visible={isDrawerVisible.value}
          mask={false}
        ></Drawer>
      </MapProvider>
    </div>
  );
}

export default App;

```
Zoom works using older version.

**Comment by dnlkoch** [comment]
Thanks for your report @jaagupku. I could reproduce your issue and it seems that there are some style defintions missing in either the the current example app or the `react-geo.css` avoiding the `map`div to get the needed height. To fix the issue just add the following block to your app, e.g. by adding it to the `index.css` file:
```

**Comment by jaagupku** [comment]
Thanks for the reply @dnlkoch. I got the map working. Also i discovered that the
[App.js](https://github.com/terrestris/react-geo-ws/blob/master/react-geo-app/src/App.js#L71) Line 71
```
map.on('postcompose', map.updateSize);
``` 
Breaks zooming in and out on openlayers version `6.1.1` and commenting it out zooming works.
Issue did not appear with OL `5.3.3`.

**Comment by annarieger** [comment]
In OpenLayers 6 `percompose` and `postcompose` events on map has been replaced by `prerender` and `postrender` (see also [release notes ol v6.0.0](https://github.com/openlayers/openlayers/releases/tag/v6.0.0)

Please try to fix the affected code line as suggested - after this it should work as expected again

**Comment by jaagupku** [comment]
Okay, I replaced `postcompose` with `postrender`
```
map.on('postrender', map.updateSize);
```
But problem still persist. I investigated it and it seems that calling `map.updateSize()` on `postrender` event makes map rerender.
```
map.on('postrender', e => {
  console.log(e, new Date().toTimeString());
  map.updateSize();
});
```
This fires `postrender` event again after rerendering. Calling `map.updateSize()` seems to cancel zoom animation and other animations (dragging map, when releasing left mouse button and map smoothly stops moving).

-------

```
map.on('postrender', e => {
  console.log(e, new Date().toTimeString());
  //map.updateSize();
});
```
^ works
